### ISSUE TITLE
Add orientation prop to `rx.stepper` component

### ISSUE THREAD

**Issue by ElijahAhianyo** [reflex-dev/reflex#2088]
`rx.stepper` component is missing an orientation prop. 

```python
class Stepper(ChakraComponent):
   ...
   orientation: Var[Literal["vertical", "horizontal"]] 
```

https://chakra-ui.com/docs/components/stepper/usage

**Comment by shu-qian** [comment]
Please assign this issue to me @ElijahAhianyo

**Comment by danik292** [comment]
@shu-qian class this please

**Comment by picklelo** [comment]
Merged in #2089
